tests/robot-tests/tests/libs/no_javascript.py

```diff
@@ -0,0 +1,15 @@
+from robot.libraries.BuiltIn import BuiltIn
+sl = BuiltIn().get_library_instance('SeleniumLibrary')
+import os
+import requests
+from bs4 import BeautifulSoup
+
+def user_checks_page_accordions_work_with_no_javascript(url):
+    response = requests.get(url)
+    soup = BeautifulSoup(response.text, "html.parser")
+    first_accordion = soup.select_one("[data-testid='accordionSection']")
+    if first_accordion.name != "div":
+        raise AssertionError('accordionSection tag should be a div')
+    first_heading = first_accordion.select_one("#publications-1-heading")
+    if first_heading.name != "div":
+        raise AssertionError('First accordion heading tag should be a div')
```

Comment:
Never do assertions on what the test id or id element tag should be. Ids should be free to be on any tag that is required to make the markup semantically correct. Asserting that it should only ever be a div is asking for brittle tests.